sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -437,6 +437,10 @@ class DataSourceV2Strategy(session: SparkSession) extends Strategy with Predicat
       val changes = keys.map(key => TableChange.removeProperty(key))
       AlterTableExec(table.catalog, table.identifier, changes) :: Nil
 
+    case AlterTableDropColumns(table: ResolvedTable, cols) =>
```

Comment:
shall we add a base trait for all alter table commands and define a `def changes: Seq[TableChanges]`? Then we can handle all alter table commands in one place.